doc/api/index.rst

```diff
@@ -21,6 +21,7 @@
    axes_api.rst
    axis_api.rst
    index_backend_api.rst
+   blocking_input_api.rst
```

Comment:
This entry is showing up a bit odd in the contents - maybe needs a different titile in `blocking_input_api.rst`?

<img width="319" alt="screen shot 2017-10-20 at 09 29 24" src="https://user-images.githubusercontent.com/6197628/31811943-478de17c-b579-11e7-89d6-2b9f77db4dea.png">

Reply:
fixed, together with a few other titles...